src/pages/Account/Admin.tsx

```diff
@@ -27,19 +31,19 @@ const VenueList: React.FC<VenueListProps> = ({ selectedVenueId }) => {
   const venues = useSelector((state) => state.firestore.ordered.venues);
   return (
     <>
-      <h4>My Venues</h4>
-      <ul>
+      <div className="page-container-adminsidebar-title">My Venues</div>
+      <ul className="page-container-adminsidebar-venueslist">
         {venues?.map((venue) => (
           <li
             key={venue.id}
-            style={venue.id === selectedVenueId ? { color: "red " } : {}}
+            className={`${selectedVenueId === venue.id ? "selected" : ""}`}
```

Comment:
dont need the template literals

Reply:
We'll also have `camp` later, I think the template litteral makes sense.